### ISSUE TITLE
[Patches] Update bigdump.php

### ISSUE THREAD

**Issue by mystralkk** [Geeklog-Core/geeklog#314]
**Reported by dhaun on 30 Apr 2010 08:49**

**Version:** 1.6.1

[Mantis Bugtracker #1143](http://project.geeklog.net/tracking/view.php?id=1143)

**Comment by mystralkk** [comment]
Related to #380

**Comment by mystralkk** [comment]
**Comment posted by Roccivic on 18 May 2010 11:41**

Patch works fine. Here&#039;s what I done:

ADDED:
	from version 0.32b 2010-03-04
	    * Change fopen mode to &quot;r&quot; as workaround for Windows issues
	from version 0.31b 2009-11-12
	    * Fixing the timezone warning for date() in PHP 5.3
	    * Add mysql module availability check
	    * Remove deprecated ereg()

NOT ADDED:
	from version 0.31b 2009-11-12
	    * Workaround for mysql_close() bug #48754 in PHP 5.3 - Was already OK?
	from version 0.30b 2009-09-02
	    * Add CSV preempty table feature - Geeklog does not handle CSV files
	    * Fix read whole line on Mac - This causes a memory leak on my pc

REMOVED:
	from version 0.28b 2007-06-08
	    * Handle CSV files (you have to specify $csv_insert_table) - Geeklog does not handle CSV files

**Comment by mystralkk** [comment]
**Comment posted by dhaun on 26 Feb 2011 21:04**

So here I was, trying to think of a GSoC project related to the install script.
Then I remembered this patch.
In trying it out, I ran into a bug with upgrading the Configuration (introduced recently, when adding the &quot;tab&quot; column).
When testing that, I noticed a missing text in the English UTF-8 language file for the Polls plugin.
When looking into that, I noticed that the language file was also missing the new $LANG_tab array.
Which made me realize that this was also missing from all the other non-English language files of all the other bundled plugins.

All fixed now :)

Oh yeah: Patch applied. Thanks for submitting it!
